src/convertToObject.js

```diff
@@ -2,17 +2,42 @@
 
 /**
  * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const sourceArray = sourceString
+    .split('\n')
+    .filter(x => x !== '' && x !== '  ;')
+    .join('').split(';').filter(x => x !== '');
+
+  const test1 = [];
+  const test2 = [];
+  const result = {};
+
+  sourceArray.forEach(element => {
+    test1.push(element.trim().split(':'));
```

Comment:
maybe the work of this trim() method will do trim() in test2 - 'test2.push(key.trim());'